src/mds/Beacon.cc

```diff
@@ -73,6 +75,7 @@ void Beacon::init(const MDSMap &mdsmap)
   _notify_mdsmap(mdsmap);
 
   sender = std::thread([this]() {
+    ceph_pthread_setname(pthread_self(), "beacon");
```

Comment:
is this just a cleanup patch or does it have some useful side-effect?

Reply:
Both. The useful side effect is that top/gdb/etc. show a proper thread name. No effect for the process itself.